gradle/download-sentry-cli.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd $(dirname "$0")
+REPO=getsentry/sentry-cli
+VERSION=1.14.0
+PLATFORMS="Darwin-x86_64 Linux-i686 Linux-x86_64 Windows-i686"
+
```

Comment:
I guess I'm being extra cautious, what if the arch names change or one is added or removed? Figured it's safe to always clean up the current state before re-download.